src/analysis/special_functions/gamma/beta.lean

```diff
@@ -507,3 +508,48 @@ end
 end real
 
 end gamma_reflection
+
+section inv_gamma
+open_locale real
+
+namespace complex
+/-! ## The reciprocal Gamma function
+
+We show that the reciprocal Gamma function `1 / Γ(s)` is entire. These lemmas show that (in this
+case at least) mathlib's conventions for division by zero do actually give a mathematically useful
+answer! (These results are useful in the theory of zeta and L-functions.) -/
+
+/-- A reformulation of the Gamma recurrence relation which is true for `s = 0` as well. -/
+lemma one_div_Gamma_eq_self_mul_one_div_Gamma_add_one (s : ℂ) :
+  1 / Gamma s = s * (1 / Gamma (s + 1)) :=
```

Comment:
I suspect that `x⁻¹` might generally be preferred to `1 / x` here; making this:
```suggestion
  (Gamma s)⁻¹ = s / Gamma (s + 1) :=
```

Reply:
What concerns me with spelling it that way is that the name of the lemma would then have to be `inv_Gamma_eq...` and it sounds like it should be about the inverse under composition of functions instead.

Reply:
I don't think the spelling matters too much.